Makefile

```diff
@@ -361,6 +361,12 @@ $(BUILD)/gomod-lint: go.mod internal/tools/go.mod common/archiver/gcloud/go.mod
 $(BUILD)/code-lint: $(LINT_SRC) $(BIN)/revive | $(BUILD)
 	$Q echo "lint..."
 	$Q $(BIN)/revive -config revive.toml -exclude './vendor/...' -exclude './.gen/...' -formatter stylish ./...
+	$Q bad="$$(find . -type f -name '*.go' -not -path './idls/*' | xargs grep -n -E '^\s*//\S' | grep -E -v '^[^:]+:[^:]+:\s*//[a-z]+:[a-z]+' || true)"; \
```

Comment:
I'd add a comment explaining this

I'm interpreting it to be selecting all relevant Go files, and then for each greppign them for comments in the pattern with whitespace followed by non-whitespace (\S), and then filtering out... I'm not sure what the intent is with that last regex

Reply:
ignore, I didn't read your PR docs

Reply:
Nah, it's legitimately weird looking unless you know these kinda esoteric details. I can comment the line above to keep intent clear without needing to read history.